pymtl3/passes/sverilog/translation/structural/SVStructuralTranslatorL2.py

```diff
@@ -112,7 +111,7 @@ def _gen_packed_array( dtype, n_dim, array ):
           assert False, "unrecognized data type {}!".format( dtype )
       else:
         ret = []
-        for i in reversed( range( n_dim[0] ) ):
+        for i in reversed( list(range( n_dim[0])) ):
```

Comment:
`reversed(range(...))` works fine, no need for an intermediate list.

(This is a running theme in Python 3 😉)